src/OpenCensus.Collector.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -62,7 +64,7 @@ public override void OnStartActivity(Activity activity, object payload)
             string path = (request.PathBase.HasValue || request.Path.HasValue) ? (request.PathBase + request.Path).ToString() : "/";
 
             ISpan span = null;
-            this.Tracer.SpanBuilderWithRemoteParent(path, SpanKind.Server, ctx).SetSampler(this.Sampler).StartScopedSpan(out span);
+            this.Tracer.SpanBuilderWithRemoteParent(path, SpanKind.Server, ctx).SetSampler(this.Sampler(new Uri(request.GetDisplayUrl()))).StartScopedSpan(out span);
```

Comment:
`sampler` may return `null` which I think should indicate to use default. So I'd suggest to have this logic implemented here - check result for `null` and fall back to the default one